libraries/botbuilder/package.json

```diff
@@ -51,12 +50,13 @@
     "node-mocks-http": "^1.16.0"
   },
   "scripts": {
-    "build": "tsc -b",
+    "build": "yarn prebuild && tsc -b",
```

Comment:
this shouldnt be necessary, it will be executed previous to the build script, since it is defined with the pre hook https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts

Reply:
Oh, you're right! Fixed. Thanks!